### PR TITLE
feat: 🎸 complete lottery put api

### PR DESCRIPTION
**Pull request by 5-michelle**
### Description:

- Notion link: https://www.notion.so/SLD-1-Server-Create-Event-API-b1fb322f949d46449f987f7f26370306?pvs=4
- complete admin lottery put

### Screenshot

截圖 2023-05-08 下午3 00 34

### CODE REVIEW

File: server/models/adminModel.js [hunk 1]
```diff
@@ -1,8 +1,92 @@
...
+          庫存張數: inventory,
+        } = item;
+
+        // error handle for excel content
+        validator.existValidate(discount_name, "discount_name", next);
+        validator.existValidate(discount_value, "discount_value", next);
+        validator.existValidate(threshold, "threshold", next);
+        validator.existValidate(inventory, "inventory", next);
+        validator.rangeValidate(discount_value, "discount_value", next);
+        validator.numberValidate(threshold, "inventory", next);
```
Comment (brandon9912) [inline server/models/adminModel.js:140]: 打錯了 "threshold"

Comment (egnkvn) [reply server/models/adminModel.js:140]: 這邊是什麼意思

**Review by egnkvn — DISMISSED** [review]
LGTM

**Review by egnkvn — APPROVED** [review]
LGTM

**Pull request merged**